lib/nrf_modem_lib/Kconfig.modemlib

```diff
@@ -62,7 +62,7 @@ config NRF_MODEM_LIB_SHMEM_TX_SIZE
 
 config NRF_MODEM_LIB_SHMEM_RX_SIZE
 	int "RX region size"
-	range 2488 32768
+	range 2616 32768
```

Comment:
I wonder if it's possible to have different ranges based on whether we are on a nRF9160 or nRF9161?

Reply:
Like so?
```
range 2616 32768 if SOC_NRF9160_SICA
range 2488 32768 if SOC_NRF9161_SICA
```

Reply:
done.